test/integrations/Elastic.Apm.AspNetCore.Tests/BodyCapturingTests.cs

```diff
@@ -285,6 +284,206 @@ public async Task ApmMiddleware_ShouldSkipCapturing_WhenInvalidContentType()
 			result.StatusCode.Should().Be(HttpStatusCode.UnsupportedMediaType);
 		}
 
+		[Fact]
+		public async Task When_CaptureBodyConfigurationToAllAndSuccessfulCall_Should_CaptureBody()
+		{
+			var sutEnv = StartSutEnv(CreateConfiguration("all"));
+
+			// build test data, which we send to the sample app
+			var data = new { Name = "John" };
+
+			var body = JsonConvert.SerializeObject(data,
+				new JsonSerializerSettings { ContractResolver = new CamelCasePropertyNamesContractResolver() });
+
+			// send data to the sample app
+			var result = await sutEnv.HttpClient.PostAsync("api/Home/Send", new StringContent(body, Encoding.UTF8, "application/json"));
+
+			// wait for the payload sender to receive the transaction
+			sutEnv.MockPayloadSender.WaitForTransactions(TimeSpan.FromSeconds(10));
+
+			// make sure the sample app received the data
+			result.StatusCode.Should().Be((HttpStatusCode)200);
+
+			// and make sure the data is captured by the agent
+			sutEnv.MockPayloadSender.FirstTransaction.Should().NotBeNull();
+			sutEnv.MockPayloadSender.FirstTransaction.Context.Request.Body.Should().Be(body);
+		}
+
+		[Fact]
+		public async Task When_CaptureBodyConfigurationToTransactionsAndSuccessfulCall_Should_CaptureBody()
+		{
+			var sutEnv = StartSutEnv(CreateConfiguration("transactions"));
```

Comment:
nit: We have constants defined for the various capture body options, e.g. https://github.com/elastic/apm-agent-dotnet/blob/main/src/Elastic.Apm/Config/ConfigConsts.cs#L134 which we could use here and in the other tests.